lms/djangoapps/certificates/migrations/0020_certificatehtmlviewconfiguration_data.py

```diff
@@ -19,7 +19,7 @@ def forwards(self, orm):
                 "company_privacy_url": "http://www.edx.org/edx-privacy-policy",
                 "company_tos_url": "http://www.edx.org/edx-terms-service",
                 "company_verified_certificate_url": "http://www.edx.org/verified-certificate",
-                "logo_src": "/static/certificates/images/logo-edx.svg",
+                "logo_src": "/static/certificates/images/logo-edx.png",
```

Comment:
NOTE: had some trouble with .svg files rendering consistently in print across browsers, so I simplified things to use .png graphics.